examples/example2/TextSourceDataModel.hpp

```diff
@@ -35,13 +35,13 @@ class TextSourceDataModel : public NodeDataModel
   bool
   captionVisible() const override { return false; }
 
-  QString
-  name() const override
+  static QString
+  Name()
   { return QString("TextSourceDataModel"); }
 
-  std::unique_ptr<NodeDataModel>
-  clone() const override
-  { return std::make_unique<TextSourceDataModel>(); }
+  QString
+  name() const override
+  { return TextSourceDataModel::Name(); }
```

Comment:
Should we be consistent with the style and use the `name` instead of `Name` ?

Reply:
The name need to be different. We may change __Name()__ to __staticName()__.

Or any other you suggest...